Base/usr/share/man/man2/module_load.md

```diff
@@ -0,0 +1,35 @@
+## Name
+
+module\_load - load a kernel module
```

Comment:
Does LibMarkdown choke on this? Ideally it would parse this the way GitHub does, i.e. auto-escape underscores inside words

Reply:
Yeah, it doesn't like it - it doesn't make the text italic or anything, but the underscore gets eaten.